.github/workflows/push.yml

```diff
@@ -0,0 +1,118 @@
+name: push-pipeline
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '**'
```

Comment:
Will this ignore all the tags, if yes then, how the tag image will be pushed after release?

Reply:
For release, we are using release.yaml workflow